src/lib/components/buttonv2/Buttonv2.component.js

```diff
@@ -0,0 +1,231 @@
+//@flow
+import React from 'react';
+import type { Node } from 'react';
+import styled, { css } from 'styled-components';
+import Tooltip from '../tooltip/Tooltip.component.js';
+import type { Props as TooltipProps } from '../tooltip/Tooltip.component.js';
+import { getTheme } from '../../utils';
+import * as defaultTheme from '../../style/theme';
+import { spacing } from '../../style/theme.js';
+
+export type Props = {
+  variant?: 'primary' | 'secondary' | 'danger' | 'outline',
+  disabled?: boolean,
+  onClick?: (any) => void,
```

Comment:
onClick takes a MouseEvent as an attribute (https://flow.org/en/docs/react/events/ , SyntheticMouseEvent<HTMLButtonElement>)

Reply:
oh. Yes. I missed that!